package/src/steps/FormView/BodyFormView.tsx

```diff
@@ -704,6 +712,27 @@ const BodyFormView: React.FC<BodyFormViewType> = (props) => {
                 className={stylesCommon["body-form-child"]}
                 label={field.humanName}
                 type={getInputType(field)}
+                onClick={()=>{
+
+                  if(getInputType(field)==="email"){
+                    const gtmData = {
+                      event: GtmEvent.ELEMENT_CLICK,
+                      action: GtmEventAction.EMAIL_FORM,
+                      category: GtmEventCategory.FIELD,
+                      label: GtmEventLabel.EMAIL_ADDRESS,
+                    };
+                    sendDataToGTM(gtmData);
+                  }
+                  if(getInputType(field)==="password"){
+                    const gtmData = {
+                      event: GtmEvent.ELEMENT_CLICK,
+                      action: GtmEventAction.EMAIL_FORM,
+                      category: GtmEventCategory.FIELD,
+                      label: GtmEventLabel.PASSWORD,
+                    };
+                    sendDataToGTM(gtmData);
+                  }
+                }}
```

Comment:
1. This logic can be moved to separate handler and use that to send crypto input event also.
2. Enum can be used instead of if condition to dynamically get the label (e.g. MenuItemLabel). Currently only if condition decides the label other logic is repeating.